tests/cypress/e2e/sign_up_create_page/create_account_button.cy.js

```diff
@@ -0,0 +1,55 @@
+import {goToLogin, gotoRegisterTeacher, goToPage, gotoRegisterStudent} from '../tools/navigation/nav.js'
+import {login, loginForUser} from '../tools/login/login.js'
+
+describe('Username field test', () => {
+  it('passes', () => {
+    goToPage(Cypress.env('register_student_page'));
+
+//      goToRegisterStudent();
+// This function is not working yet. Preferably we use the goToRegisterStudent function instead of the goToPage function.
```

Comment:
I just asked @TiBiBa and he set the PR to draft because of this comment. Can you remove the dead (commented) code and use the right function (or open an issue explaining the work that still needs to be done)

Reply:
Thankyou for the review. Will look into it!